site/content/en/docs/Installation/Install Agones/helm.md

```diff
@@ -221,12 +221,12 @@ The following tables lists the configurable parameters of the Agones chart and t
 | `gameservers.podPreserveUnknownFields`                   | Disable [field pruning][pruning] and schema validation on the Pod template for a [GameServer][gameserver] definition                                                                                                    | `false`                            |
 | `helm.installTests`                                      | Add an ability to run `helm test agones` to verify the installation                                                                                                                                                     | `false`                            |
 
-{{% feature publishVersion="1.29.0" %}}
+{{% feature publishVersion="1.30.0" %}}
 **New Configuration Features:**
 
 | Parameter                             | Description                                                                                                                                                                                                                           | Default |
 |---------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------|
-| `agones.image.registry`                                  | Global image registry for all the Agones system images                                                                                                                                                                                   | `us-docker.pkg.dev/agones-images/release`
+| `agones.image.registry`                                  | Global image registry for all the Agones system images                                                                                                                                                                                                     | `us-docker.pkg.dev/agones-images/release`
```

Comment:
This still needs to be removed from here, and then merged with the table above.

Reply:
Here's an example from a previous release:
https://github.com/googleforgames/agones/pull/2852/files#diff-481b1839587cfc3a359e5e82a78b375445ebe75fe39d5c070150bc4f3a2f51adR169

Reply:
Table can be there without any records. is that correct?